### DISCUSSION TITLE
Unable to register telegram

### DISCUSSION THREAD

**Discussion by lirs1990** [community/community#125063] [New to GitHub]
How to bypass the prompt to add an email address to get the mobile phone card verification code to log in，No need to verify by previously logged in mobile phone number

**Comment by Malith-Rukshan** [comment]
Hey there,

It sounds like you're running into some challenges with Telegram registration.  Here are a couple of things to try:

**Check Telegram's Official Support**: Telegram has comprehensive documentation on the registration process. It might offer some specific solutions or troubleshooting steps: https://telegram.org/faq

**Community Forums**: You might find similar discussions or potential workarounds in Telegram's official forums or other online communities focused on privacy and security.

**Important Note**:
It's generally not recommended to bypass security features like email verification. These steps are in place to protect your account and data. If you're having trouble receiving the verification code, make sure your phone number is entered correctly and you have a stable network connection.
